lib/clover_field.cpp

```diff
@@ -134,9 +133,10 @@ namespace quda {
   void CloverField::setTuningString()
   {
     LatticeField::setTuningString();
-    int aux_string_n = TuneKey::aux_n / 2;
-    int check = snprintf(aux_string, aux_string_n, "vol=%lu,precision=%d,Nc=%d", volume, precision, nColor);
-    if (check < 0 || check >= aux_string_n) errorQuda("Error writing aux string");
+    std::stringstream aux_ss;
```

Comment:
fmt again?

Reply:
like
```
aux_str = fmt::format("vol={},precision={},Nc={}",volume, precision, nColor);
```

Reply:
If we decide to add that I can take care of that.

Reply:
Yes, that would be a nice clean up.  You can add that to your soon-to-be-filed issue  😛

Reply:
There is an old issue #1176